modules/account-lib/test/unit/coin/dot/transactionBuilder/unnominateBuilder.ts

```diff
@@ -0,0 +1,103 @@
+import should from 'should';
+import { UnnominateBuilder } from '../../../../../src/coin/dot';
+import * as DotResources from '../../../../resources/dot';
+import { buildTestConfig } from './base';
+
+describe('Dot Unnominate Builder', () => {
+  let builder: UnnominateBuilder;
+
+  const sender = DotResources.accounts.account1;
+  const refBlock = '0x149799bc9602cb5cf201f3425fb8d253b2d4e61fc119dcab3249f307f594754d';
+  const genHash = '0xe143f23803ac50e8f6f8e62695d1ce9e4e1d68aa36c1cd2cfd15340213f3423e';
+  const specVersion = 9150;
+
+  beforeEach(() => {
+    const config = buildTestConfig();
+    builder = new UnnominateBuilder(config);
+  });
+
+  describe('build unnominate transaction', () => {
+    it('should build a unnominate transaction', async () => {
+      builder
+        .sender({ address: sender.address })
+        .validity({ firstValid: 3933, maxDuration: 64 })
+        .referenceBlock(refBlock)
+        .sequenceId({ name: 'Nonce', keyword: 'nonce', value: 200 })
+        .fee({ amount: 0, type: 'tip' })
+        .version(8);
+      builder.sign({ key: sender.secretKey });
+      const tx = await builder.build();
+      const txJson = tx.toJson();
+      should.deepEqual(txJson.sender, sender.address);
```

Comment:
are there any properties on the tx or txJson that can verify that a `chill` txn was built?
you may need to add properties from the tx to txJson by adding a check here https://github.com/BitGo/BitGoJS/blob/c259b9d815da67c7e21cda59b53412417d27e3ee/modules/account-lib/src/coin/dot/transaction.ts#L100

Reply:
For chill, there are no arguments that need to be passed in so there really isn't any chill specific properties. I'm not quite sure how else to verify it is a chill transaction.

Reply:
Have you checked what the `tx` object looks like? But yeah I see what you're saying, if there are no args then I doubt they'd show up in the `tx` object. It might be worth double checking tho. It may hold a method name or something `chill` specific.

Reply:
I double checked, the only thing unique is the method name, and this should be tested in 'should build from raw unsigned/signed tx' tests because it calls validateDecodedTransaction which checks method name.

Reply:
thats true. alrighty - looks like this needs a rebase to resolve conflicts. everything else looks good to me. nice work!